content/blog/deploying-mysql-schemas-using-dynamic-providers/index.md

```diff
@@ -0,0 +1,284 @@
+---
+title: "Deploying a MySQL schema using Dynamic Providers"
+date: 2020-08-16T15:06:26-07:00
+meta_desc: Leveraging Pulumi Dynamic Providers to expand opportunities in cloud architecture design
+meta_image: meta.png
+authors: ["vova-ivanov"]
+tags: ["aws", "python", "mysql"]
+---
+
+Previously, we created a Python voting application using Flask and Redis. In this blog post, we will explore how to create a MySQL database and initialize it with a schema and some data. What seems to be a simple step is much more interesting than it appears, however, because Pulumi does not have a built-in library for such a task. To do it, we will create our own library: a Dynamic Provider.
```

Comment:
```suggestion
Previously, we created a Python voting application using Flask and Redis. In this blog post, we will explore how to create a MySQL database and initialize it with a schema and data. What seems to be a simple step is much more interesting than it appears, because Pulumi's MySQL provider does not support creating and populating tables. To do it, we will extend it with a Dynamic Provider.
```

Reply:
Updated introduction with edits.